src/main/java/org/sert2521/gamename/Drivetrain/Drivetrain.kt

```diff
@@ -0,0 +1,51 @@
+package org.sert2521.gamename.Drivetrain
+
+import edu.wpi.first.wpilibj.command.Command
+import edu.wpi.first.wpilibj.command.Subsystem
+import org.sert2521.gamename.FRONT_LEFT_MOTOR
+import org.sert2521.gamename.FRONT_RIGHT_MOTOR
+import org.sert2521.gamename.OI
+import org.sert2521.gamename.REAR_LEFT_MOTOR
+import org.sert2521.gamename.REAR_RIGHT_MOTOR
+import org.strongback.components.Motor
+import org.strongback.drive.TankDrive
+import org.strongback.hardware.Hardware
+
+/**
+ * This is the Drivetrain subsystem, which is in charge of controlling the speeds of the individual
+ * talons and converting joystick input to movement speeds.
+ */
+object Drivetrain : Subsystem() {
```

Comment:
Wait. Are you saying have all the commands and stuff in the root, too? Aren't we organizing stuff based on the subsystem this year, like:

```
- gamename
  - drivetrain
    - Drivetrain.kt
    - commands
        - ArcadeDrive.kt
        - TankDrive.kt
        - DriveForward.kt
        - TurnDegrees.kt
```

Or are you just saying we should move things from the `...gamename.drivetrain.commands` package to the `...gamename.drivetrain` package? I just did that because I like the separation between commands and subsytems, but either way works for me.